.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     strategy:
-      matrix:  # Support latest and one minor back
-        go: ['1.15', '1.16']
+      matrix: # Support latest and one minor back
+        go: ["1.16", "1.17"]
```

Comment:
Just for the record this pulls the latest patch release for each minor, which is what we wan't but wasn't obvious to me.

Reply:
Yes. I double checked the logs to make sure it did that because I wasn't sure either.